src/responder/kcm/kcmsrv_ops.h

```diff
@@ -24,6 +24,7 @@
 
 #include "config.h"
 
+#include <dhash.h>
```

Comment:
I propose to use forward declaration instead of header inclusion.

Reply:
Why?

Reply:
I would put it other way: no reason to include a header (if one can go with a forward declaration).

Example reasoning could be: https://stackoverflow.com/a/9906429

But I don't insist.

Reply:
Hmm, this makes sense. However, I see a problem in the fact that hash_table_t is defined using typedef so I would have to declare it as:

```
struct hash_table_str;
typedef struct hash_table_str hash_table_t;
```

which seems wrong. What do you think?

Reply:
Funny thing is, it currently compiles without any forward declaration :) Just because `kcmsrv_ops.c` includes dhash.h before kcmsrv_ops.h anyway, and during compilcation of kcmsrv_cmd.c compiler seems to do some optimizations.

You are right, with typedef this is more tricky.

Let's leave it as it is and I will mark it as resolved.